### ISSUE TITLE
Some genes not showing

### ISSUE THREAD

**Issue by lpalbou** [geneontology/noctua-form#122]
From our discussion this morning with @tmushayahama.

The first activities don't have genes:
http://noctua-dev.berkeleybop.org/workbench/noctua-form/?model_id=gomodel%3A59a9023b00000151

But they do in the graph:
http://noctua-dev.berkeleybop.org/editor/graph/gomodel:59a9023b00000151?

Example: 
"suppression by virus of host MAVS activity" enabled by "UniProtKB:Q69027" is in the graph, but not the form.
![Screen Shot 2020-11-04 at 9 37 53 AM](https://user-images.githubusercontent.com/24249870/98149091-fa118a00-1e81-11eb-95fb-324930af02d5.png)

![Screen Shot 2020-11-04 at 9 42 18 AM](https://user-images.githubusercontent.com/24249870/98149286-0b5a9680-1e82-11eb-8692-a285e4a81369.png)

This is affecting both prod and dev.

**Comment by lpalbou** [comment]
After further investigation, it seems the root-type is missing from the barista response:

![Screen Shot 2020-11-04 at 9 43 25 AM](https://user-images.githubusercontent.com/24249870/98149788-86bc4800-1e82-11eb-9e3e-124caa04607f.png)

amigo/golr do resolve to something: http://amigo.geneontology.org/amigo/gene_product/UniProtKB:P08617 but minerva is now using the refactored neo, so there may be some differences.

I will investigate a bit more with Patrick to see if he entered the gene id in free text rather than autocomplete.

**Comment by lpalbou** [comment]
If this one could be due to the annotation to covid protein (and larger discussion/issue), we also have that issue for a human protein. From the same model (http://noctua.geneontology.org/workbench/noctua-form/?model_id=gomodel:59a9023b00000151&barista_token=qfmsf7j12312v0ihr17j), the activity ubiquitin-protein transferase activity has an enabler "UniProtKB:Q5JSK4" who shows up in the graph, but not in noctua form (no root-type) and this one is not in amigo/golr.

**Comment by kltm** [comment]
`UniProtKB:Q69027` This is not found in NEO and does not seem to be in the upstreams:
ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/goa_human.gpi.gz
ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/goa_human_complex.gpi.gz
ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/goa_human_isoform.gpi.gz
ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/goa_human_rna.gpi.gz
ftp://ftp.pantherdb.org/downloads/paint/presubmission/gene_association.paint_human.gaf.gz

**Comment by lpalbou** [comment]
It could be that Patrick used the unreviewed ID of RNF128 instead of the reviewed one http://amigo.geneontology.org/amigo/gene_product/UniProtKB:Q8TEB7 which is indexed in GOLr. The covid protein X was however in golr/amigo.

**Comment by pgaudet** [comment]
@pmasson55 had asked to load for viral entries, as far as I know this has not been done yet. 

@kltm may know where this stands.

**Comment by goodb** [comment]
FYI, whenever you see the uniprot id of the gene and not the label in the noctua graph editor, its an indication that minerva does not actually know anything about the gene.

**Comment by pmasson55** [comment]
Hi all ,

You re looking at the RLR pathway which is not COVID related, It's a general pathway where I have annotated viral proteins from different viruses inhibiting this pathway (I think it's one of my first models that I made more than 2 years ago) . The protein you talk about is HBX from hepatitis B virus, which is indeed not in NEO.  Concerning RNF128 you're right, the swissprot should have been used and not a TREMBL. I can fix it.
